docs/bdd/index.rst

```diff
@@ -0,0 +1,24 @@
+The BDD Guide
+=============
+
+Behaviour driven development is an approach to software development process that provides software development and management teams
```

Comment:
Shouldn't we always write `Behaviour Driven Development` with the capital letter at the beginning?

Reply:
In a sentence it is not needed I suppose.